renderlib/Logging.h

```diff
@@ -1,5 +1,8 @@
 #pragma once
 
+// remove after boost fixes issue in boost 1.78

```

Comment:
Should this be a TODO, or maybe have a date added?  Or even a url to the issue tracker?